lib/prefab/caching_http_connection.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+module Prefab
+  class CachingHttpConnection
+    CACHE_SIZE = 2.freeze
```

Comment:
File this under "More confusing ruby constant behavior," but since literal numbers are immutable, the freeze here has no effect.

rubocop sez:
> Style/RedundantFreeze: Do not freeze immutable objects, as freezing them has no effect.